blue-note/theme.json

```diff
@@ -0,0 +1,224 @@
+{
+	"$schema": "https://schemas.wp.org/wp/6.2/theme.json",
+	"customTemplates": [
+		{
+			"name": "single-no-sidebar",
+			"postTypes": [
+				"page",
+				"post"
+			],
+			"title": "Single, no sidebar"
+		}
+	],
+	"settings": {
+		"appearanceTools": true,
+		"color":{
+			"palette": [
+				{
+					"slug": "base",
+					"name": "Base",
+					"color": "#F5F3EA"
+				},
+				{
+					"slug": "contrast",
+					"name": "Contrast",
+					"color": "#000000"
+				},
+				{
+					"slug": "primary",
+					"name": "Primary",
+					"color": "#000000"
```

Comment:
I think we can remove the repeated color and consider the naming of the other two. This theme was designed before TT3 and was using a different convention, I believe it's best if we follow the direction of TT3 instead